### PR TITLE
trial

### PR DESCRIPTION
**Pull request by ravindersaluja**
trying new recipes

### CODE REVIEW

**Comment by yeshapatel356** [comment]
...

**Comment by balagitaccount** [comment]
done

File: pizza.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+
+Recipe from Navin Reddy
+
+
+1. Go to Dominos.com
+2. Select the pizza.
+3. Order
```
Comment (Chaitanya-Varanasi) [inline pizza.txt:8]: lol

**Review by Chaitanya-Varanasi — COMMENTED** [review]
try to use a real word application

**Comment by aashirathore** [comment]
biryani............

**Comment by IKiriZudoI** [comment]
Comment

**Pull request closed without merging**